docs/content/guides/author-apps/kubernetes/howto-yaml-base-support/snippets/basemanifest.bicep

```diff
@@ -0,0 +1,43 @@
+import radius as radius
+
+
+@description('Specifies the image of the container resource.')
+param magpieimage string = 'ghcr.io/radius-project/dev/magpiego:pr-498da3203f'
```

Comment:
Please change this from a parameter to a hard-coded value in the Radius container

Reply:
Also let's find another image than magpie. Would the demo image work?